### PR TITLE
content card example

### PR DESCRIPTION
**Pull request by davidbielik**
content card example for web

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/web/content_cards/customization/custom_ui.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ description: "This article covers components of creating a custom UI for your we
 
 > This article covers components of creating a custom UI for your web application.
 
-## Refreshing the feed
+## Refreshing the feed {#refresh}
```
Comment (spvismaya) [inline _docs/_developer_guide/platform_integration_guides/web/content_cards/customization/custom_ui.md:16]: what does `{#refresh}` do here exactly? I saw the .md file and it just shows as a normal heading with {#refresh} next to it. I am not very familiar with this

File: _docs/_developer_guide/platform_integration_guides/web/content_cards/customization/custom_ui.md [hunk 2]
```diff
@@ -36,29 +36,47 @@ Content Cards will only refresh on session start if `subscribeToContentCardsUpda
...
+    }
+  })
 });
 
 braze.openSession();
 ```
 
 ## Logging events
 
+{% alert info %}
```
Comment (ttsaiveerappan) [inline _docs/_developer_guide/platform_integration_guides/web/content_cards/customization/custom_ui.md:61]: Should this be `note` or `warning` or `important`?

Comment (davidbielik) [reply _docs/_developer_guide/platform_integration_guides/web/content_cards/customization/custom_ui.md:61]:
```suggestion
{% alert note %}
```

File: _docs/_developer_guide/platform_integration_guides/web/content_cards/customization/custom_ui.md (hunk 1 above)
Comment (davidbielik) [reply _docs/_developer_guide/platform_integration_guides/web/content_cards/customization/custom_ui.md:16]: this lets you control the hash `#refresh` at the end of the docs URL to link directly to this header

**Review by spvismaya — APPROVED** [review]

**Review by josh-mccrowell-braze — APPROVED** [review]

**Pull request merged**